src/main/java/uk/gov/digital/ho/pttg/api/ArchiveResource.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.digital.ho.pttg.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.pttg.ArchiveService;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.digital.ho.pttg.application.LogEvent.*;
+
+@RestController
+@Slf4j
+public class ArchiveResource {
+
+    private final ArchiveService archiveService;
+
+    public ArchiveResource(ArchiveService archiveService) {
+        this.archiveService = archiveService;
+    }
+
+    @GetMapping(value = "/archived", produces = APPLICATION_JSON_VALUE)
+    public List<ArchivedResult> getArchivedResults(
+            @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate fromDate,
+            @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate toDate
+    ) {
+        log.info("Request received for archived results between {} and {}", fromDate, toDate,
+                value(EVENT, PTTG_AUDIT_ARCHIVE_REQUEST_RECEIVED));
```

Comment:
The event sounds like it is a request to create an archive... maybe just changing ARCHIVE_REQUEST to GET_ARCHIVED_REQUEST or similar?

Reply:
Agreed